lib/api/model/events.dart

```diff
@@ -797,13 +797,13 @@ class UpdateMessageEvent extends Event {
   @JsonKey(includeToJson: true)
   String get type => 'update_message';
 
-  final int? userId; // TODO(server-5)
-  final bool? renderingOnly; // TODO(server-5)
+  final int userId;
```

Comment:
Discussion on this `userId` change: [#api documentation > &#96;user_id&#96; in &#96;update_message&#96; event @ 💬](https://chat.zulip.org/#narrow/channel/412-api-documentation/topic/.60user_id.60.20in.20.60update_message.60.20event/near/2215174)